src/CabanaPD_Particles.hpp

```diff
@@ -601,36 +594,13 @@ class Particles<MemorySpace, LPS, TemperatureIndependent, Dimension>
         _timer.stop();
     }
 
-    void output( [[maybe_unused]] const int output_step,
-                 [[maybe_unused]] const double output_time,
-                 [[maybe_unused]] const bool use_reference = true )
+    template <typename... OtherFields>
+    void output( const int output_step, const double output_time,
```

Comment:
Why is the formatting for this output different from the one for PMB, NoOutput?

Reply:
The base output allows unused since there may be no HDF5 turned on in the end